library/lib/build.js

```diff
@@ -106,14 +109,21 @@ module.exports = function build({ watch }) {
         publicPath,
         libraryTarget: 'commonjs2'
       },
-      externals: nodeExternals({ whitelist: ['@kaliber/config', kaliberBuildClientModules, /\.css$/] }),
+      externals: [
+        nodeExternals(externalConfForModulesDir('node_modules')),
+        yarnWorkspaceDir && nodeExternals(externalConfForModulesDir(path.resolve(yarnWorkspaceDir, 'node_modules')))
+      ].filter(Boolean),
```

Comment:
This probably wouldn't work correctly. I'm not sure what the returntype of `nodeExternals` is, if its an array we need `...`

Reply:
Hmm, the webpack documentation is not clear about this part. The library returns a function. I'm not sure if webpack can handle an array here. We might need to create a function to do the combining.

This be tested quite easily though:
- Make sure the `example` directory has no `node_modules` and try to build with the `example` directory as `cwd`
- Have normal (non workspace) application and run it build there as well

Reply:
This seems to be the correct way according to https://github.com/liady/webpack-node-externals/pull/40#issuecomment-356648523